scripts/scratch.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
can we keep scratch files out of the repo? There's also `scripts/test-util/1.sh` that does this exact liquid stake

Reply:
Meant to add this to .gitignore, thx for pointing this out